web/src/SidebarResources.tsx

```diff
@@ -112,6 +216,9 @@ function SidebarGroupedByLabels(props: SidebarProps) {
         />
       ))}
       <SidebarLabelListSection {...props} label="unlabeled" items={unlabeled} />
+      <SidebarListSection name="Tiltfile">
```

Comment:
nit - if the user types something that doesn't match into the resource name filter box, this will still show the "tiltfile" section w/ no resources. This is inconsistent with the label groups, which disappear when no resources match.

Reply:
ahh, thank you! i forgot to test with the resource filter. let me take a look and fix.

Reply:
this should be fixed now, so that we don't display the grouped view when a resource name filter is applied.